__docs__/wonder-blocks-dropdown/single-select.argtypes.ts

```diff
@@ -16,15 +28,15 @@ const argTypes: ArgTypes = {
     labels: {
         control: {type: "object"},
         description:
-            "The object containing the custom labels used inside this component.",
+            "The object containing the custom values used inside this component.",
         table: {
             type: {summary: "Labels"},
         },
     },
     showOpenerLabelAsText: {
         control: {type: "boolean"},
-        description: `When false, the SelectOpener can show a Node as a label. When true, the
-         SelectOpener will use a string as a label. If using custom OptionItems, a
+        description: `When false, the SelectOpener can show a Node as a value. When true, the
+         SelectOpener will use a string as a value. If using custom OptionItems, a
```

Comment:
I find this description a bit confusing (it might just be me!). Do you have any ideas on how we could make it easier to understand? 

Maybe something like this?
```suggestion
        description: `When false, the SelectOpener can show a Node as the selected value. When true, the
         SelectOpener will use a string as the selected value. If using custom OptionItems, a
```

(same for MultiSelect)

Reply:
Such a great call out. This part IS uniquely confusing since the API changed from "labels" to "values". I'll also admit this particular prop was confusing to me from the get-go. 😅 

I think `selected value` works. It might not totally fix the confusing prop name, though?

Reply:
I think the docs will help! In the context of form fields that have labels, it is confusing 😆 I think renaming the prop will be a bigger lift since we'll want to make this prop consistent with all the components that have translations too!